src/structs/default.md

```diff
@@ -0,0 +1,34 @@
+# Setting Default values in Structs
+
+You can either derive or define default values for structs.  
+
+```rust,editable
+#[derive(Debug)]
+struct Person {
+    name: String,
+    age: u8,
+}
+impl Default for Person {
+  fn default() -> Person { Person { name:"Bot".to_string(), age:0 } }
+}
+impl Person {
+    fn new(name: String, age: u8) -> Person {
+        Person { name, age }
+    }
+}
+
+fn main() {
+    let peter = Person::new(String::from("Peter"), 27);
+    println!("{peter:?}");
+    
+    let tmp = Person {..Default::default()};
+    println!("{tmp:?}");
+    
+    let tmp = Person {age:12, .. Default::default()};
```

Comment:
Please note that we try to format everything the way `rustfmt` would do it:

```suggestion
    let tmp = Person { age: 12, ..Default::default() };
```

Reply:
I removed this file and moved content to Speaker notes.